WalletWasabi/WabiSabi/Backend/Models/Alice.cs

```diff
@@ -31,5 +33,13 @@ public void SetDeadlineRelativeTo(TimeSpan connectionConfirmationTimeout)
 			// Have alice timeouts a bit sooner than the timeout of connection confirmation phase.
 			Deadline = DateTimeOffset.UtcNow + (connectionConfirmationTimeout * 0.9);
 		}
+
+		private uint256 CalculateHash()
+			=> StrobeHasher.Combine(new()
+			{
+				{ nameof(Coin.TxOut), Coin.TxOut.ToHex() },
+				{ nameof(Coin.Outpoint), Coin.Outpoint.ToHex() },
+				{ nameof(OwnershipProof), OwnershipProof.ToHex() },
```

Comment:
here too we should probably string constants instead of relying on `nameof`, that way if anybody refactors anything it won't accidentally result in a protocol break (possibly a very annoying one to debug, since the hashes won't match with no clear indication of why)

Reply:
Done.